src/GitVersionTask.Tests/AssemblyInfoBuilderTests.cs

```diff
@@ -30,13 +31,21 @@ public void VerifyCreatedCode()
         Approvals.Verify(assemblyInfoText);
 
         var compilation = CSharpCompilation.Create("Fake.dll")
-            .WithOptions(new CSharpCompilationOptions(OutputKind.NetModule))
+            .WithOptions(new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary))
             .AddReferences(MetadataReference.CreateFromFile(typeof(object).Assembly.Location))
             .AddSyntaxTrees(CSharpSyntaxTree.ParseText(assemblyInfoText));
-        
 
-        var emitResult = compilation.Emit(new MemoryStream());
-        Assert.IsTrue(emitResult.Success, string.Join(Environment.NewLine, emitResult.Diagnostics.Select(x => x.Descriptor)));
+
+        using (var stream = new MemoryStream())
+        {
+            var emitResult = compilation.Emit(stream);
+            
+            Assert.IsTrue(emitResult.Success, string.Join(Environment.NewLine, emitResult.Diagnostics.Select(x => x.Descriptor)));
```

Comment:
Yea, it's not my assert. I thought I'd just leave as much code I didn't write untouched to focus the PR on one thing so it would be easier to review. :smile: